internal/provider/provider.go

```diff
@@ -95,25 +101,33 @@ func (p *IncusProvider) Schema(_ context.Context, _ provider.SchemaRequest, resp
 				NestedObject: schema.NestedBlockObject{
 					Attributes: map[string]schema.Attribute{
 						"name": schema.StringAttribute{
-							Required:    true,
-							Description: "Name of the Incus remote. Required when incus_scheme set to https, to enable locating server certificate.",
+							Optional:    true,
```

Comment:
We should make the `name` required, otherwise it could lead to possible misconfiguration as the name of a remote is always mandatory in the CLI.